src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -238,7 +205,7 @@
   <data name="IdRequirements" xml:space="preserve">
     <value>Id must be any combination of upper or lower case ASCII letters ('A'..'Z', and 'a'..'z'), numerals ('0'..'9'), '-' and '.', with a length limit of 64 characters. (This might be an integer, an un-prefixed OID, UUID, or any other identifier pattern that meets these constraints.)</value>
   </data>
-  <data name="IfMatchHeaderRequiredForResource" xml:space="preserve">
```

Comment:
It appears my editor has removed some spacing from this file, unfortunatley.  I will fix.